### ISSUE TITLE
Some keyboard navigation commands doesn't work in notebooks

### ISSUE THREAD

**Issue by thibaultdvx** [niivue/ipyniivue#289]
Hey, first of all thank you for the great work!

I'm talking about the navigation commands (`k`, `j`, `k`, `l`, `ctrl + u`, `ctrl + v`). I'm using the Jupyter extension (`2025.9.1`) of VSCode (`1.123.0`). In my conda environement I have `ipyniivue` (`2.4.4`) and `ipykernel` (`6.29.5`).

Looks like the commands (`h`, `ctrl + u` and `ctrl + v`) are working when navigating between slices, but not (`j`, `k`, `l`). The latter seem to be overshadowed by Jupyter keyboard commands.

Best,
Thibault

**Comment by hanayik** [comment]
@thibaultdvx , thanks for reporting. I am very surprised to see someone other than myself using those Niivue navigation commands :). I initially added them to the core Niivue library because it was useful during development, and they were so esoteric that the chance of keyboard conflicts was low. 

I do admit, I do not use Jupyter notebooks at all, so keeping these Niivue navigation commands active in that environment was not explicitly tested for command conflicts. This was an oversight that we can correct in the core Niivue package and then propagate to ipyniivue. 

I think the best solution is to make these configureable to the user in the following ways:
1. Users of Niivue and ipyniivue should be able to set the key commands they wish to use for navigation
2. Users of Niivue and ipyniivue should be able to completely disable and enable the navigation commands with a simple boolean flag such as `isKbNavigationEnabled`. 

Would this solve the issue for you (and likely other notebook users)?

cc @cdrake

**Comment by thibaultdvx** [comment]
Hey @hanayik, these commands are very useful to get a precise slice. I think your solution is great!
